src/components/sidebarContent/TestData.vue

```diff
@@ -31,32 +34,41 @@
 import SidebarCard from '@/components/wrappers/SidebarCard.vue';
 import ButtonText from '@/components/formElements/ButtonText.vue';
 import ErrorMessage from '@/components/textElements/ErrorMessage.vue';
-import { mapActions } from 'vuex';
+import InfoMessage from '@/components/textElements/InfoMessage.vue';
+import { mapGetters, mapActions } from 'vuex';
 import axios from 'axios';
 
 export default {
   components: {
     SidebarCard,
     ButtonText,
     ErrorMessage,
+    InfoMessage,
   },
   data: () => ({
     areTestListsShown: false,
     testData: [],
     isRequestProcessing: false,
     errorMessage: '',
   }),
+  computed: {
+    ...mapGetters({
+      lists: 'lists',
+    }),
+  },
   created() {
     axios.get('/test_data.json').then(({ data }) => {
       this.testData = data;
     });
+    this.areTestListsShown = Boolean(!this.lists.length);
```

Comment:
!! instead of Boolean
if you want

Reply:
hm, no after !this.lists.length it's already Boolean. ! is an operator which return Boolean